.travis.yml

```diff
@@ -69,5 +69,6 @@ deploy:
   upload-dir: openweave-nrf52840-lock-example/$TRAVIS_BUILD_NUMBER
   local-dir: "$TRAVIS_BUILD_DIR/build_artifacts"
   on:
+    branch: master
```

Comment:
@jaylogue , do we require CI builds from branches other than master to be deployed ? If yes, I can revert this change.

Reply:
branch builds should include master branch also
unless we only want to have master branch build deployed, this condition will work for it.